Jenkinsfile

```diff
@@ -191,6 +191,23 @@ pipeline {
                 }
             }
         }
+        stage('Check PR') {
```

Comment:
I don't really want to get more emails out of CI, so it would be good to merge this one with the checkpatch job/email, or if not then put it in github actions so it's not consuming valuable space in the Jenkinsfile.

Initially I would also be happy to land this PR without this feature in order for us to test stability before a widespread push for adoption.

Reply:
This should not result in an e-mail.  It's simply a message added to the steps of the stage which will be yellow, but the build will not fail because of it.  It's hoped that the yellow stage will at least make people curious and look to see why it's yellow.